recipes/qdldl-python/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.1.3" %}
+
+package:
+  name: qdldl-python
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/q/qdldl/qdldl-{{ version }}.tar.gz
+  sha256: 2e98bfcc43de55225b82f5895f8266e79a305b174490c60bc559c1538f1be920
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - numpy
+    - pybind11
+    - scipy
```

Comment:
Actually, are you linking against scipy?

Reply:
https://github.com/oxfordcontrol/qdldl-python/blob/master/cpp/wrapper.cpp#L43 and https://github.com/oxfordcontrol/qdldl-python/blob/master/cpp/wrapper.cpp#L68 sure look like it...

I'm gonna try switching to `run: {{ pin_compatible('scipy') }}`. Not sure what the minimum `scipy` version is to build `qdldl-python`? @bstellato @gbanjac

Reply:
So scipy builds with `1.2.1` by default, and uses the newest (on CF) `1.4.1` for run. That sounds alright IMO.

Reply:
Yes we are using `scipy`. Not sure what the minimum compatible version is, but the default  runs without problems.

Reply:
using scipy at runtime is different than linking against it shared libraries (which is rather uncommon)

Reply:
The C++ is loading scipy at runtime using pybind11. It is the same as importing a module in python. We are not linking against it.

Reply:
OK, then it shouldn't be in host

Reply:
Fixed.
@scopatz PTAL :)